Mathlib/CategoryTheory/Bicategory/Adjunction.lean

```diff
@@ -92,8 +92,6 @@ namespace Adjunction
 
 attribute [simp] left_triangle right_triangle
 
--- attribute [local simp] leftZigzag rightZigzag
```

Comment:
Can we find the source of this? Did it originally make some proofs nicer, or was it just mistakenly committed